modules/expectations/mobile_expectations.lua

```diff
@@ -0,0 +1,42 @@
+local expectations = require('expectations')
+local events = require('events')
+
+local Expectation = expectations.Expectation
+local Event = events.Event
+local SUCCESS = expectations.SUCCESS
+local FAILED = expectations.FAILED
+
+local module = {}
+local mt = { __index = { } }
+
+function mt.__index:ExpectEvent(event, name)
+  local ret = Expectation(name, self.session.connection)
+  ret.event = event
+  event_dispatcher:AddEvent(self.session.connection, event, ret)
+  self.session.exp_list:Add(ret)
+  return ret
+end
+
+
+function mt.__index:ExpectAny()
+  local event = events.Event()
+  event.level = 1
```

Comment:
@VVeremjova what is event level? Why it is 1? Is it moved code? 
Please add comment of create enum for event levels or remove usage of event levels from there